app/controllers/task_controller.rb

```diff
@@ -30,12 +30,45 @@ def delete
   	redirect_to action: "index"
   end
 
+  def edit
```

Comment:
Based on the rails naming convention, I think you want this controller to be plural `TasksController`